elevate-cpanel

```diff
@@ -2965,6 +2965,8 @@ EOS
 
         cpev::update_stage_file( { 'mysql-version' => $mysql_version } );
 
+        File::Copy::copy( $cnf_file, "$cnf_file.rpmsave_pre_elevate" ) or WARN("Couldn't backup $cnf_file to $cnf_file.rpmsave_pre_elevate: $!");
```

Comment:
I think we should move not copy this so there's no confusion if the rpmsave file is about the recent RPM removal?